fetch.bs

```diff
@@ -6608,12 +6622,11 @@ method steps are:
   <a for=Request>signal</a>:
 
   <ol>
-    <li><p>Set <var>locallyAborted</var> to true.
+   <li><p>Set <var>locallyAborted</var> to true.
 
-    <li><p><a>Abort fetch</a> with <var>p</var>, <var>request</var>, and
-    <var>responseObject</var>.
+   <li><p><a>Abort fetch</a> with <var>p</var>, <var>request</var>, and <var>responseObject</var>.
```

Comment:
I'm not sure if this works as at this point _responseObject_ is null and later it might get a value at which point we'd want this to operate on that value I think. Is there a better way to tackle this?

Reply:
Could you check the signal right before creating the response object?